main.py

```diff
@@ -375,6 +375,7 @@ def index(marker=None, message=None):
     context['pref_radius'] = pref_radius
     today = datetime.date.today()
     context['default_end_date_format'] = today.strftime('%Y-%m-%d')
+    context['default_start_date_format'] = (today - datetime.timedelta(days=700)).strftime('%Y-%m-%d')
```

Comment:
I see. That's because you don't know the length of a year without knowing which year it is. But wouldn't 730 be more accurate (it's really two years for most years)?